api/api.go

```diff
@@ -21,5 +22,10 @@ func Setup(ctx context.Context, r *mux.Router, dataStore DataStore) *API {
 
 	r.HandleFunc("/mongocheck", api.AddDataSets(ctx)).Methods("POST")
 	r.HandleFunc("/mongocheck", api.GetDataSets(ctx)).Methods("GET")
+
+	// TODO: implement read/write endpoints here.
```

Comment:
It would be good to add a reference to the ticket, so that it's clear that we have a plan for this implementation.